### PR TITLE
New website

### PR DESCRIPTION
**Pull request by xlingyy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,143 +1,82 @@
 <!DOCTYPE html>
-<html lang="">
-  <head>
-    <link href="https://fonts.bunny.net/css2?family=Fira+Sans&display=swap" rel="stylesheet">
```
Comment (moehreag) [inline index.html:4]: You might want to keep this line to ensure the font loads correctly if it isn't installed on the user's system (depending on which font you want the website to use)

**Comment by moehreag** [comment]
I think that's the only note (since you kept the font in the css, but not the required line in the html)

File: index.html (hunk 1 above)
Comment (xlingyy) [reply index.html:4]: Switched to Fira Sans

**Review by moehreag — APPROVED** [review]
looks good to me!

**Pull request merged**